internal/config/config.yaml

```diff
@@ -0,0 +1,9 @@
+HTTP:
+  PORT: ":8083"
+
+DB:
+  HOST: "localhost"
+  PORT: 55660
+  USERNAME: "testuser"
+  PASSWORD: "password"
+  NAME: "testdb"
```

Comment:
A couple of things here:

- We need to expose something like the HTTP API port through a CLI flag
- The database credentials can live as a Kubernetes Secret, and we can instruct users to point the curator stack to the Secret resource $somehow (although that $somehow is likely a singleton CRD that drives configuration to the rest of the curator stack)